packages/devextreme-cli/testing/testing.js

```diff
@@ -0,0 +1,86 @@
+const jest = require('jest');
+const reactEnv = require('./env.react');
+const vueEnv = require('./env.vue');
+const angularEnv = require('./env.angular');
+const minimist = require('minimist');
+const path = require('path');
+const fs = require('fs');
+const Emmiter = require('events');
+
+const args = minimist(process.argv.slice(), {
+    default: {
+        template: 'all'
+    },
+    alias: {
+        t: 'template'
+    }
+});
+
+const envs = {
+    'react': reactEnv,
+    'angular': angularEnv,
+    'vue': vueEnv
+};
+
+let killerEvent = new Emmiter();
+let done = Object.keys(envs).length;
+killerEvent.on('kill', function() {
```

Comment:
Why do we need separate event for this. What if it will be a method with the same `done` counter?